collateral/deployment/k8s-without-helm-reference.md

```diff
@@ -0,0 +1,53 @@
+# Deploying Sourcegraph to Kubernetes (k8s) without Helm
+
+## Introduction
+This material is intended to be used as a handout to customers working with the Customer Engineering team. The materials are meant to serve as an overview and resource to aid in deep diving into Sourcegraph's documentation and common deployment tasks.
+
+**Table of Contents:**
+
+- [Introduction](#introduction)
+- [Sourcegraph Setup and Deployment](#sourcegraph-setup-and-deployment)
+	- [Prerequisites](#prerequisites)
+  - [Create your cluster](#create-your-cluster)
+  - [Configure your manifest file](#configure-your-manifest-file)
+  - [Install Sourcegraph](#install-sourcegraph)
+- [Next Steps](#next-steps)
+
+## Sourcegraph Setup and Deployment
+
+### Prerequisites
+- Ensure you have your Sourcegraph Enterprise License
+- Access to deploy Kubernetes clusters
+  - Minimum Kubernetes version: `v1.19` and `kubectl` `v1.19`
+  - Support for Persistent Volumes (SSDs recommended)
+- Determine resource requirements to properly size your instance: [Resource Estimator](https://docs.sourcegraph.com/admin/deploy/resource_estimator)
+- Install relevant CLIs for your provider of choice:
+  - AWS: [`aws` cli](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) and [`ejsctk`](https://eksctl.io/introduction/#installation)
+  - Azure: [`az` cli](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+  - GCP: [`gcloud` cli](https://cloud.google.com/sdk/gcloud)
+
+### Create your cluster
+- Follow the instructions for your provider of choice to create your Kubernetes (k8s) cluster: [Cloud installation instructions](https://docs.sourcegraph.com/admin/deploy/kubernetes#cloud-installation)
```

Comment:
@mike-r-mclaughlin Are the values suggested in this doc good or no?

Reply:
If they aren't, let's PR the original doc so we have a good single source of truth. 🙏

Reply:
I think the issue is more that the appropriate nodes to use vary deployment-by-deployment (https://stackoverflow.com/c/sourcegraph/questions/187/188#188 has more context on AWS node choice), but I defer to Mike.

Reply:
Will totes wait for @mike-r-mclaughlin, thanks!

I do think regardless if the information isn't correct in our official documentation we need a better way to show that in the documentation. This is to reference the official docs, not be a separate documentation piece (or a second source of truth).